blacksmith/experiments/jax/distil_bert/test_distil_bert_flax.yaml

```
@@ -26,12 +26,12 @@ alpha_ce: 1.0
 alpha_cos: 0.1
 
 # Logging
-use_wandb: False
+use_wandb: True
 experiment_name: "Flax DistilBERT on SST-2"
-project_name: "bert-distillation"
+project_name: "DP bert-distillation in Flax"
```

nit: Keep project_name either as dashed or as words.

```suggestion
project_name: "dp-bert-distillation-flax"
```